protocol/x/clob/keeper/liquidations.go

```diff
@@ -67,6 +76,12 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 
 		liquidationOrders = append(liquidationOrders, *liquidationOrder)
 	}
+	telemetry.MeasureSince(
+		startGetLiquidationOrders,
```

Comment:
LMK if you all think this metric name is a misnomer, since I know this doesn't technically "get" the liquidation orders since they're recreated when placing them.

Reply:
should we just modify the sorting latency below to include this? since this is only used for sorting

Reply:
Agreed, created an M6 ticket [here](https://linear.app/dydx/issue/CLOB-963/break-up-liquidatesubaccountsagainstorderbook-steps-into-functions) to consolidate this logic. LMK if you think it should be M5 and I can move it up + potentially take it on immediately